stripped/colabs/wybranie_danych_z_jednego_mies_stripped.py

```diff
@@ -0,0 +1,102 @@
+"""
+<a href="https://colab.research.google.com/github/PiotrMaciejKowalski/BigData2024Project/blob/Narzdzie-do-wizualizacji-na-mapach/wybranie_danych_z_jednego_mies.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>
+"""
+
+!apt-get install openjdk-8-jdk-headless -qq > /dev/null
+!wget -q dlcdn.apache.org/spark/spark-3.5.0/spark-3.5.0-bin-hadoop3.tgz
+!tar xf spark-3.5.0-bin-hadoop3.tgz
+!pip install -q findspark
+
+import os
+os.environ["JAVA_HOME"] = "/usr/lib/jvm/java-8-openjdk-amd64"
+os.environ["SPARK_HOME"] = "/content/spark-3.5.0-bin-hadoop3"
+
+import findspark
+findspark.init()
+
+from pyspark.sql import SparkSession
+
+from google.colab import drive
+
+from pyspark.sql.types import IntegerType, FloatType, StringType, StructType
+
+import pandas as pd
+
+
+from pyspark.sql import Window
+from pyspark.sql.functions import row_number
+
+
+"""
+# Wczytanie zbioru danych
+"""
+
+spark = SparkSession.builder\
+        .master("local")\
+        .appName("Colab")\
+        .config('spark.ui.port', '4050')\
+        .getOrCreate()
+
+
+drive.mount('/content/drive')
+
+
+# Wczytanie zbioru sampled w celu pobrania nazw kolumn
+sampled = pd.read_csv('/content/drive/MyDrive/sampled_NASA_200k.csv')
```

Comment:
To nie jest scieżka do wspólnego zasobu dyskowego (czyli notatnik nie zadziała nikomu poza Tobą)

Reply:
poprawione